Jenkinsfile

```diff
@@ -6,6 +6,8 @@ pipeline {
     timeout(time: 2, unit: "HOURS")
   }
   environment {
+    REG = "cloud-native-image-registry.westus.cloudapp.azure.com/"
+    TAG = "devel"
```

Comment:
We can rely on the `TAG` being correct for master (`TAG?=devel`) and release (`TAG?=vX.Y.Z`) branches. The only place we'd want to override `TAG` is when we push pull requests for testing. In that case the `TAG` needs to be something like `CHANGE_ID-rejected`.

Since we don't need to push images for testing right now, `TAG` must be left out.